e2e/test/scenarios/dashboard/dashboard-visualizer.cy.spec.js

```diff
@@ -0,0 +1,1532 @@
+
+    H.modal().within(() => {
+      // Turn into a pie chart
+      cy.findByTestId("viz-picker-main").icon("pie").click();
+      assertDataSourceColumnSelected(
+        ORDERS_COUNT_BY_PRODUCT_CATEGORY.name,
+        "Count",
+      );
+      assertDataSourceColumnSelected(
+        ORDERS_COUNT_BY_PRODUCT_CATEGORY.name,
+        "Product → Category",
+      );
+      pieMetricWell().findByText("Count").should("exist");
+      pieDimensionWell().findByText("Product → Category").should("exist");
+      H.echartsContainer().findByText("18,760").should("exist"); // total value
+
+      // Turn into a funnel
+      cy.findByTestId("viz-picker-main").icon("funnel").click();
+      assertDataSourceColumnSelected(
+        ORDERS_COUNT_BY_PRODUCT_CATEGORY.name,
+        "Count",
+      );
+      assertDataSourceColumnSelected(
+        ORDERS_COUNT_BY_PRODUCT_CATEGORY.name,
+        "Product → Category",
+      );
+      verticalWell().findByText("Count").should("exist");
+      horizontalWell().within(() => {
+        cy.findByText("Product → Category").should("exist");
+        cy.findByText("Doohickey").should("exist");
+        cy.findByText("Gadget").should("exist");
+        cy.findByText("Gizmo").should("exist");
+        cy.findByText("Widget").should("exist");
+        cy.findAllByTestId("well-item").should("have.length", 5);
+      });
+    });
+  });
+
+  describe("cartesian charts", () => {
+    it("should allow to change viz settings", () => {
+      createDashboardWithVisualizerDashcards();
+      H.editDashboard();
+
+      // Cartesian chart
+      H.findDashCardAction(H.getDashboardCard(0), "Edit visualization").click();
+      H.modal().within(() => {
+        cy.button("Settings").click();
+
+        H.goalLine().should("not.exist");
+        vizSettingControl("Goal line")
+          .findByRole("switch")
+          .click({ force: true });
+        H.goalLine().should("exist");
+
+        // TODO Fix multi-series chart settings and extend the test
+        chartLegend().findByText("Series B").should("not.exist");
+        cy.findAllByTestId("series-name-input")
+          .eq(1)
+          .type("{selectall}{del}Series B")
+          .blur();
+        chartLegend().findByText("Series B").should("exist");
+
+        cy.button("Save").click();
+      });
+      H.getDashboardCard(0).within(() => {
+        H.goalLine().should("exist");
+        chartLegend().findByText("Series B").should("exist");
+      });
+
+      // Pie chart
+      H.findDashCardAction(H.getDashboardCard(2), "Edit visualization").click();
+      H.modal().within(() => {
+        cy.button("Settings").click();
+        cy.findByText("Display").click();
+
+        H.echartsContainer().within(() => {
+          cy.findByText("200").should("exist");
+          cy.findByText("TOTAL").should("exist");
+        });
+        vizSettingControl("Show total")
+          .findByRole("switch")
+          .click({ force: true });
```

Comment:
Does it require `"force": true`? I see pie chart tests use `cy.findByText("Show total").click();`